modules/social_features/social_content_block/modules/social_event_content_block/src/Plugin/ContentBlock/EventContentBlock.php

```diff
@@ -74,6 +77,15 @@ public function query(SelectInterface $query, array $fields) {
               $range['end'] = new \DateTime('last day of next month 23:59');
               break;
 
+            case 'ongoing':
+              $today = new \DateTime();
```

Comment:
@rollki since we don't use `$today` variable in other places, but just new `\DateTime()`, I think we need to be consistent here as well or replace it to `$today` everywhere.

Reply:
Fixed. Variable was removed.